src/settings.py

```diff
@@ -2,24 +2,34 @@
 
 
 from typing import Dict
-from pydantic import BaseSettings, HttpUrl
+from pydantic import BaseSettings, HttpUrl, AnyHttpUrl
 
 
 class AggregatorSettings(BaseSettings):
     """Create Settings from env."""
 
-    snyk_package_url_format: HttpUrl = "https://snyk.io/vuln/{ecosystem}:{package}"
+    license_analysis_base_url: AnyHttpUrl = "http://f8a-license-analysis:6162"
     snyk_signin_url: HttpUrl = "https://snyk.io/login"
+    snyk_package_url_format: HttpUrl = "https://snyk.io/vuln/{ecosystem}:{package}"
     snyk_ecosystem_map: Dict[str, str] = {"pypi": "pip"}
-    disable_unknown_package_flow: bool = False
 
 
 class RecommenderSettings(BaseSettings):
     """Create Recommender Settings from env."""
 
-    chester_service_host: str = "notset"
-    pypi_service_host: str = "notset"
-    maven_service_host: str = "notset"
-    service_port: int = 6006
+    npm_insights_base_url: AnyHttpUrl = "http://f8a-npm-insights:6006"
```

Comment:
Why are these hard coded?

Reply:
This shall be overridden through env variable(NPM_INSIGHTS_BASE_URL). It just holds the default value. Refer template.yaml changes.